cclib/parser/turbomoleparser.py

```diff
@@ -983,6 +983,150 @@ def extract(self, inputfile, line):
             
             self.append_attribute("etdips", [tdm_x, tdm_y, tdm_z])
             
+        # Excitation energies with ricc2.
+        #  +================================================================================+
+        #  | sym | multi | state |          CC2 excitation energies       |  %t1   |  %t2   |
+        #  |     |       |       +----------------------------------------+--------+--------+
+        #  |     |       |       |   Hartree    |    eV      |    cm-1    |    %   |    %   |
+        #  +================================================================================+
+        #  | a   |   1   |   1   |    0.3257471 |    8.86403 |  71493.234 |  94.89 |   5.11 |
+        #  | a   |   1   |   2   |    0.3257471 |    8.86403 |  71493.232 |  94.89 |   5.11 |
+        #  | a   |   1   |   3   |    0.3864541 |   10.51595 |  84816.880 |  95.05 |   4.95 |
+        #  | a   |   1   |   4   |    0.3938325 |   10.71673 |  86436.241 |  94.09 |   5.91 |
+        #  | a   |   1   |   5   |    0.3938325 |   10.71673 |  86436.241 |  94.09 |   5.91 |
+        #  | a   |   1   |   6   |    0.4122171 |   11.21700 |  90471.202 |  93.32 |   6.68 |
+        #  | a   |   1   |   7   |    0.4351537 |   11.84114 |  95505.195 |  94.16 |   5.84 |
+        #  | a   |   1   |   8   |    0.4454969 |   12.12259 |  97775.278 |  94.26 |   5.74 |
+        #  | a   |   1   |   9   |    0.4454969 |   12.12259 |  97775.273 |  94.26 |   5.74 |
+        #  | a   |   1   |  10   |    0.5036295 |   13.70446 | 110533.909 |  93.51 |   6.49 |
+        #  +================================================================================+
+        # TODO: Perhaps need a more general way to look for lines like this?

```

Comment:
The only thing I can think of right now would be a regex with `(?:(CC2|ADC\(2\))) excitations energies\s+|` (untested :) ) but I don't think it's worth making that change yet.